SetDemo/.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
You should add .gitignore to your project folder before `git add` all files of the project so that git knows what to ignore. These files in .idea should not be a part of a pull request.